stage1/init/common/pod.go

```diff
@@ -427,6 +427,20 @@ func generateDeviceAllows(root string, appName types.ACName, mountPoints []types
 	return devAllow, nil
 }
 
+// supportsNotify return true if in the pod manifest appc.io/executor/supports-systemd-notify is set to true
```

Comment:
Does it still look in the pod manifest as the comment says, or look in the image manifest as the code below suggests?